eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlResource.java

```diff
@@ -61,6 +68,7 @@ public EidasSamlParserResponse post(@Valid EidasSamlParserRequest request) throw
         ProxyNodeLogger.addContext(ProxyNodeMDCKey.EIDAS_ISSUE_INSTANT, authnRequest.getIssueInstant().toString());
         ProxyNodeLogger.addContext(ProxyNodeMDCKey.EIDAS_LOA, getLevelsOfAssurance(authnRequest));
         ProxyNodeLogger.info("Authn request validated by ESP");
+        SUCCESSFUL_REQUESTS.labels(authnRequest.getIssuer().getValue()).inc();
```

Comment:
Do we want to count total requests as well as successful requests? That way, you can compute a rate of successes.